javascript/packages/orchestrator/src/providers/k8s/kubeClient.ts

```diff
@@ -67,8 +67,8 @@ export class KubeClient extends Client {
 
   constructor(configPath: string, namespace: string, tmpDir: string) {
     super(configPath, namespace, tmpDir, "kubectl", "kubernetes");
-    this.configPath = configPath;
-    this.namespace = namespace;
+    this.configPath = process.env.KUBECONFIG || configPath;
+    this.namespace = process.env.ZOMBIE_NAMESPACE || namespace;
```

Comment:
```suggestion
    this.configPath = process.env.KUBECONFIG || configPath;
```
Same here, and I think we can also move this _or_ the the orchestrator layer.

Reply:
done!